### DISCUSSION TITLE
This is really great!

### DISCUSSION THREAD

**Discussion by anorthall** [elsbrock/hetzner-radar#41] [General]
Thanks for the effort. Great UI and very useful. Really appreciate it.

**Comment by anorthall** [comment]
Would you consider making the history of JSON available? I'd be interested in seeing if there is any pattern in the pricing, and whether you can predict the cheapest price a server will reach (and therefore attempt to wait for it).

**Reply by elsbrock** [reply]
Thank you! The JSON is available on the https://github.com/elsbrock/hetzner-radar/tree/data branch.

**Reply by anorthall** [reply]
Wonderful! Thanks!
